### PR TITLE
[enhancement][xs] Added `.rdf` files access

### LINKED ISSUE #2: Add RDF file extension to the white list
Using `noanonaccess` along with DCAT extension will prevent the datasets to show at Google Data Search Engine and others. 

## Acceptance Criteria

* [ ] Being able to access a RDF file even when logged out of CKAN. Which allow Google crawlers to find the datasets

## Tasks

* [ ] Add `*.rdf` to the whitelist

## Analysis

`noanonaccess` already have a whitelist structure. 
https://github.com/datopian/ckanext-noanonaccess/blob/master/ckanext/noanonaccess/plugin.py#L20

We need to create a path that can receive any dataset that ends with extension `.rdf`

### PR DESCRIPTION
**Pull request by gavram**
@anuveyatsu 

This fixes #2 

Please, review this.
This fix/enhancement enables `rdf` files access without login.

This closes #2

### CODE REVIEW

File: ckanext/noanonaccess/plugin.py [hunk 1]
```diff
@@ -19,6 +19,8 @@ def __call__(self, environ, start_response):
             return self.app(environ,start_response)
         elif '/uploads/' in environ['PATH_INFO'] or '/download/' in environ['PATH_INFO']:
             return self.app(environ,start_response)
+        elif '.rdf' in environ['PATH_INFO']:
```
Comment (anuveyatsu) [inline ckanext/noanonaccess/plugin.py]: @gavram can we do "ends with" instead of "in"?

**Review by anuveyatsu — CHANGES_REQUESTED** [review]

File: ckanext/noanonaccess/plugin.py (hunk 1 above)
Comment (gavram) [reply ckanext/noanonaccess/plugin.py]: Yes we can. Will do it.

Comment (gavram) [reply ckanext/noanonaccess/plugin.py]: @anuveyatsu 

Done

**Review by anuveyatsu — APPROVED** [review]
LGTM

**Pull request merged**